src/utilities.js

```diff
@@ -107,13 +127,15 @@ export function getResults(searchType, obj) {
     },
   };
 
+  // Create an array of keywords to query
+  let comparisonItem = formatKeywords(obj);
```

Comment:
Now we don't need the `comparisonItem` variable here either, just plug `formatKeywords(obj)` right into the `JSON.stringify(`.

So get rid of this line